bin/gencode

```diff
@@ -16,6 +16,9 @@ if [[ -n $check ]]; then
     echo Checking gencode docs links...
     bin/gencode_docs check_links
 
+    echo Checking inline examples
+    bin/gencode_doc_examples check
```

Comment:
put in bin/gencode_docs, not bin/gencode  --> the general flow would be:

`bin/gencode` --> `bin/gencode_docs` --> `bin/gencode_docs_examples`